public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Code-Type-WebApp</title>
```

Comment:
Different Issues needs to be addressed in different PR's.
Somebody has fixed this in previous PR. You can take a fresh pull and ignore this change.

Reply:
Okay got it.
This are some of my initial contributions, so making such a mistakes.
Thanks for the pinpointing my mistakes :)